Base/Logger/Logger.Static/FileLogger.hpp

```diff
@@ -2,22 +2,16 @@
 
 #include <Logger/ILogger.hpp>
 #include <Utility/Utility.hpp>
-#include <algorithm>
-#include <chrono>
-#include <condition_variable>
```

Comment:
As we discussed IRL, this header actually uses std::condition_variable. Other headers can be checked using CI.